app/controllers/attendances_controller.rb

```diff
@@ -13,7 +13,7 @@ def destroy
   private
 
     def attendance
-      current_user.attendances.find(params[:id])
+      current_user.student_team.attendances.find(params[:id])
```

Comment:
@klappradla  I saw this controller being used just when a user (now the team) has to confirm her participation in the conference (after choosing the preferences). So, theoretically, just the students from the team should have access to them. The details around this will be developed better in this issue: https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues/772e.

We have included the student_team in the private method attendance because the attendance belongs to the team, not to the user. But this page couldn't be visited by any one who is not a student from that specific team.

I think that this can change with the issue #772. What do you think?